Code/Editor/CryEditDoc.h

```diff
@@ -60,21 +52,11 @@ class CCryEditDoc
     int GetModifiedModule();
 
     // Return level path.
-    // If a slice is being edited then the path to a placeholder level is returned.
     QString GetLevelPathName() const;
 
-    // Set path of slice or level being edited.
+    // Set path of level being edited.
     void SetPathName(const QString& pathName);
 
-    // Return slice path, if slice is open, an empty string otherwise. Use GetEditMode() to determine if a slice is open.
-    QString GetSlicePathName() const;
-
-    // Return The current type of document being edited, a level or a slice
-    // While editing a slice, a placeholder level is opened since the editor requires an open level to function.
-    // While editing a slice, saving will only affect the slice and not the placeholder level.
-    DocumentEditingMode GetEditMode() const;
-
-    // Return slice path if editing slice, level path if editing level.
     SANDBOX_API QString GetActivePathName() const;
```

Comment:
Seems like this function can also be deleted

Reply:
Will add to the follow-ups, thank you :D